docs/developer-docs/gas-cost.md

```diff
@@ -1,100 +1,230 @@
+Additionally, gas fees on Ethereum can fluctuate in price unpredictably, resulting in drastically different gas fees based on the current price of ETH and the current network conditions. This can make using dapps on Ethereum difficult and expensive, as one day it may cost significantly more than the day prior, or vice versa. Since ICP uses cycles, which have a fixed price based on XDR, the price is predictable and stable.
 
-Cost per Transaction in USD (XDR/USD exchange rate as of November 23, 2022).
+One downside of the ICP cycles model in comparison to the EVM gas model is that it requires prerequisite steps and ongoing maintenance for developers. ICP tokens must be converted into cycles, then cycles must be deposited into a canister's cycles balance. Canisters must have their cycles balances maintained and regularly topped up as they continuously use resources; otherwise, the canister will be removed from the network if it runs out of cycles. However, a [freezing threshold](/docs/current/tutorials/developer-journey/level-1/1.6-managing-canisters#setting-the-canisters-freezing-threshold) can be set that pauses a canister's executions if the cycles amount is expected to fall below a certain amount, and there are several community tools that have been developed to automate managing a canister's cycles, such as [CycleOps](https://cycleops.dev/).
 
-The following table shows the calculated storage cost per GB for a 30-day month: 
 
-|                      |                                    | 13-node Application Subnets | 34-node Application Subnets |
-|----------------------|------------------------------------|-----------------------------|-----------------------------|
-| GB Storage Per Month | For storing a GB of data per month | $0.431                      | $1.127                      |
+## Canister calls
+
+A canister call on ICP refers to invoking a function within a smart contract. Calls can come in the form of messages, executions, data storage, or they can use special features such as HTTPS outcalls, chain-key signing cryptography, and the Bitcoin API integration.
+
+At a high level, canister calls can be visualized using the following diagram:
+
+![Canister calls overview](./_attachments/transaction-overview.png)
+
+Each type of canister call has a different cycles cost associated with it, and the canister responsible for paying the cycles varies based on the type of call.
+
+### Canister creation
+
+An initial fee is charged when a canister is first created. Canisters are created by deploying a project onto an instance of the IC replica (locally or on the mainnet), or a canister can be created by another canister that is already running on a replica instance. Canisters created by another canister are referred to as 'child canisters'. The cycles charge for creating a canister is charged to the canister that is being created.
```

Comment:
This paragraphs sounds verbose. Wdyt:

"Canister creation costs N cycles or approximately $X. Canisters can be created by users or other canisters."

Other information in the paragraph is probably not relevant for person searching for prices.

Reply:
Revised to "Canister creation costs 100B cycles or approximately $0.133661 USD. Canisters can be created by users or other canisters."